packages/storage/src/adapters/S3Storage/S3UrlSigner.ts

```diff
@@ -0,0 +1,22 @@
+import { UrlSigner } from '../../UrlSigner.js'
+
+import type { S3Storage } from './S3Storage.js'
+export type S3SignedUrlSettings = {
+  storage: S3Storage
+}
+
+export class S3UrlSigner extends UrlSigner {
+  private storage: S3Storage
```

Comment:
We generally prefer to use JavaScript style `#private` if possible because that makes it a proper hard private class member.

Reply:
I copied:

```ts
export class UrlSigner {
  private secret: string
  private endpoint: string

  constructor({ secret, endpoint }: SignedUrlSettings) {
    this.secret = secret
    this.endpoint = endpoint
```

Do both need to be changed @Tobbe ?

Reply:
Nah, we can leave it for now and then fix it all later.